main.go

```diff
@@ -1,99 +1,39 @@
 package main
 
 import (
-	"bytes"
-	"encoding/json"
 	"flag"
-	"net/http"
-	"time"
-
-	v1 "github.com/openshift/api/route/v1"
+	routeshandler "github.com/code-ready/routes-controller/pkg/routes-handler"
```

Comment:
This needs to be routeshandler `"github.com/crc-org/routes-controller/pkg/routes-handler"` after commit 5cc302fd4